tanka-util/README.md

```diff
@@ -2,28 +2,30 @@
 permalink: /
 ---
 
-# package helm-util
+# package tanka-util
 
 ```jsonnet
-local helm-util = import "github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet"
+local tanka-util = import "github.com/grafana/jsonnet-libs/tanka-util/main.libsonnet"
```

Comment:
`local tanka-util` requires an underscore.

Reply:
This is autogenerated by docsonnet :grimacing:

Reply:
But it is invalid jsonnet, so needs fixing :-)